libsrc/telestrat/chline.s

```diff
@@ -0,0 +1,22 @@
+;
+; jede jede@oric.org 2018-04-17
+;
+
+; void chline (unsigned char length);
+;
+
+        .export         _chline
+        .include        "telestrat.inc"
+        .include        "zeropage.inc"        
+
+
+.proc _chline
+    sta     tmp1
+@loop:
+    lda     #'-'            ; Horizontal line screen code
```

Comment:
In other areas you changed comments to start lowercase. So maybe this one should start lowercase in the first place.

Reply:
Fixed sorry for the problem